src/v2/Apps/Conversation/Components/TimeSince.tsx

```diff
@@ -35,9 +35,38 @@ const exactDate = (time: string) => {
   }
 }
 
+const minutesSinceDate = (time: string | DateTime): number => {
+  if (!time) {
+    return null
+  }
+  const date = typeof time === "string" ? DateTime.fromISO(time) : time
+  return Math.floor(Math.abs(date.diffNow("minutes").toObject().minutes))
+}
+
 const relativeDate = (time: string) => {
   if (!time) return null
-  return DateTime.fromISO(time).toRelative()
+  const date = DateTime.fromISO(time)
+  const minutesSince = minutesSinceDate(date)
+
+  if (minutesSince <= 1) {
```

Comment:
Let me know what you think here - I tried to keep all handling of formatting contained close together within one function for explicitness.

`relativeDate` will always accept a timestamp and return a formatted string.